tools/grunt-tasks/chrome/bridge.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * grunt-contrib-qunit
```

Comment:
Copied from grunt-contrib-qunit, this is a support file that it uses to hook into puppeteer.